bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/GemsGuideStarWorker.java

```diff
@@ -276,6 +330,18 @@ private GemsGuideStars findGuideStars(ObsContext obsContext, Set<edu.gemini.spMo
         }
     }
 
+    public static Option<GemsGuideStars> findGuideStars(final ObsContext obsContext) throws Exception {
+        try {
+            final Set<edu.gemini.spModel.core.Angle> posAngles = getPosAngles(obsContext);
+            final List<GemsCatalogSearchResults> results = unloggedSearch(GemsGuideStarSearchOptions.DEFAULT, GemsTipTiltMode.canopus, obsContext, posAngles,
+                    scala.Option.empty());
+            final List<GemsGuideStars> gemsResults = GemsResultsAnalyzer.instance().analyze(obsContext, posAngles, results, scala.Option.empty());
```

Comment:
I'll ask Andy and Bryan to test this thoroughly, but the preliminary results seem to be consistent with what we expect from what I can tell.

Thanks for checking this for me: it is appreciated!